internal/server/v1/info/variable_group.go

```diff
@@ -52,9 +52,16 @@ func BulkVariableGroupInfo(
 	store *store.Store,
 	cache *resource.Cache,
 ) (*pb.BulkVariableGroupInfoResponse, error) {
-	// TODO(shifucun): handle "nodes" in request, now always return all the
-	// varible group data.
+	// TODO (shifucun):
+	// The response here is directly from the raw SVG group, not consistent
+	// with the VariableGroupInfo() API.
+	// Ideally, both APIs need to filter out the child variable (group) that has
+	// no data, but this is indicated with a "has_data" field, to
+	// accomocate the "Show all statistical variables" in UI widget. The UI
+	// should call this API twice, w/o constrained_entities to achieve that.
```

Comment:
For my own understanding, for now VariableGroupInfo does filter out child variable groups with no data, but BulkVariableGroupInfo does not. Is that  correct?

Reply:
Yes, the behavior is not consistent now. I would like both to behave like the documentation.